webpack.config.js

```diff
@@ -164,11 +203,9 @@ module.exports = {
             loader: 'css-loader',
             options: {
               importLoaders: 2,
-              url: (_url, resourcePath) => {
-                // only resolve URLs for dependencies
-                return resourcePath.includes('node_modules');
-              },
-            }
+              url: filterCssImport,
+              import: filterCssImport,
```

Comment:
```suggestion
              import: filterCssImport,
              sourceMap: true,
```
Otherwise debugging Fomantic styling is nearly impossible